src/centrality/betweenness.c

```diff
@@ -105,6 +105,8 @@ static int igraph_i_betweenness_cutoff_weighted(
         igraph_real_t minweight = igraph_vector_min(weights);
         if (minweight <= 0) {
             IGRAPH_ERROR("Weight vector must be positive", IGRAPH_EINVAL);
+        } else if (igraph_is_nan(minweight)) {
+            IGRAPH_ERROR("Weight vector must not contain NaN values", IGRAPH_EINVAL);
```

Comment:
Won't we get a misleading warning here if there are no edges in the graph? In that case, the weight vector is empty, so the min of the vector is NaN, while technically it does not _contain_ any NaN values.

Reply:
Sorry, my bad, I did not notice the `no_of_edges > 0` condition.